pkg/user/handler_test.go

```diff
@@ -135,6 +136,53 @@ func TestHandler_SignIn(t *testing.T) {
 	tokenService.AssertExpectations(t)
 }
 
+func TestHandler_SignIn_GetTokensError(t *testing.T) {
+	id := uint(1)
+	email := "someone@something.org"
+	password := "passwordpasswordpasswordpassword"
+	errorMessage := "some err"
+
+	c, err := config.New()
+	assert.NoError(t, err)
+
+	userService := &mockUserService{}
+	userService.
+		On("SignIn", email, password).
+		Return(&model.User{
+			Model:    gorm.Model{ID: id},
+			Email:    email,
+			Password: password,
+		})
+	tokenService := &mockTokenService{}
+	tokenService.
+		On("GetTokens", mock.AnythingOfType("*model.User"), "").
+		Return(nil, errors.New(errorMessage))
+
+	r := gin.Default()
+	authentication := middleware.NewAuthentication(userService, tokenService)
+	r.Use(middleware.ErrorHandler(), authentication.BasicAuthentication)
+	handler := NewHandler(c, userService, tokenService)
+	r.POST("/tokens", handler.SignIn)
+
+	recorder := httptest.NewRecorder()
+
+	req, err := http.NewRequest(http.MethodPost, "/tokens", nil)
+	assert.NoError(t, err)
+	req.Header.Set("Content-Type", "application/json; charset=UTF-8")
+	req.SetBasicAuth(email, password)
+
+	r.ServeHTTP(recorder, req)
+
+	actual := recorder.Code
```

Comment:
to me 

`assert.Equal(t, http.StatusInternalServerError, recorder.Code)`

reads better than putting the `Code` into its own var that only lives for two lines